lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -70,6 +70,12 @@ class GoogleCloudOutput < BufferedOutput
     # component (Docker, Kubelet, etc.) logs from container logs.
     config_param :detect_subservice, :bool, :default => true
 
+    # The regular expression to use on Kubernetes logs to extract some basic
+    # information about the log source. The regex must contain capture groups
+    # for pod_name, namespace_name, and container_name.
+    config_param :kubernetes_tag_regexp, :string, :default =>
+      '\.(?<pod_name>[^_]+)_(?<namespace_name>[^_]+)_(?<container_name>.+)$'
```

Comment:
It's not the default being used in our fluentd-gcp config, since there we're using the record reformer to strip that stuff out of the tag:
https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/fluentd-gcp/fluentd-gcp-image/google-fluentd.conf